worlds/messenger/__init__.py

```diff
@@ -116,44 +210,132 @@ def create_items(self) -> None:
             total_seals = min(len(self.multiworld.get_unfilled_locations(self.player)) - len(itempool),
                               self.options.total_seals.value)
             if total_seals < self.total_seals:
-                logging.warning(f"Not enough locations for total seals setting "
-                                f"({self.options.total_seals}). Adjusting to {total_seals}")
+                logging.warning(
+                    f"Not enough locations for total seals setting "
+                    f"({self.options.total_seals}). Adjusting to {total_seals}"
+                )
                 self.total_seals = total_seals
             self.required_seals = int(self.options.percent_seals_required.value / 100 * self.total_seals)
 
             seals = [self.create_item("Power Seal") for _ in range(self.total_seals)]
-            for i in range(self.required_seals):
-                seals[i].classification = ItemClassification.progression_skip_balancing
             itempool += seals
 
+        self.multiworld.itempool += itempool
         remaining_fill = len(self.multiworld.get_unfilled_locations(self.player)) - len(itempool)
         if remaining_fill < 10:
             self._filler_items = self.random.choices(
-                                      list(FILLER)[2:],
-                                      weights=list(FILLER.values())[2:],
-                                      k=remaining_fill
+                list(FILLER)[2:],
+                weights=list(FILLER.values())[2:],
+                k=remaining_fill
```

Comment:
originally this did just call `get_filler()` and `get_filler_item_name` had a larger arbitrary value because i usually need to create somewhere around 50-80 items here. berserker commented that it was wasteful to make my cache that large though so i opted to just not use the cache here so it doesn't have to constantly rebuild. the main reason i make the cache at all is because even though that only really happens if core calls it, core is almost never going to call your `get_filler()` only a single time and i'd rather not do the random calculation every single time it gets called.